### ISSUE TITLE
function not found

### ISSUE THREAD

**Issue by minhtq1207** [pytti-tools/pytti-core#211]
i got this issue with PYTTI on colab, please check 
![image](https://user-images.githubusercontent.com/7424881/175256801-0a7a694f-02f6-4425-8eda-6ce622bb324c.png)

here is my setting for above :

SETTINGS:
{'defaults': ['_self_', {'conf': '_empty'}], 'scenes': 'heavy rain in the city | kaleidoscope | made of crystals | over the cloud | Rose Quartz | ', 'scene_prefix': '', 'scene_suffix': '', 'direct_image_prompts': '', 'init_image': '', 'direct_init_weight': '', 'semantic_init_weight': '', 'image_model': 'Unlimited Palette', 'vqgan_model': 'imagenet', 'animation_mode': 'off', 'width': 1024, 'height': 768, 'steps_per_scene': 200, 'steps_per_frame': 25, 'interpolation_steps': 0, 'learning_rate': 0.2, 'reset_lr_each_frame': False, 'seed': -8854551145582748953, 'cutouts': 40, 'cut_pow': 2, 'cutout_border': 0.25, 'border_mode': 'clamp', 'field_of_view': 60, 'near_plane': 1, 'far_plane': 10000, 'input_audio': '', 'input_audio_offset': 0, 'input_audio_filters': [], 'pre_animation_steps': 0, 'lock_camera': True, 'translate_x': '-1700*sin(radians(1.5))', 'translate_y': '0', 'translate_z_3d': '(50+10*t)*sin(t/10*pi)**2', 'rotate_3d': '[cos(radians(1.5)), 0, -sin(radians(1.5))/sqrt(2), sin(radians(1.5))/sqrt(2)]', 'rotate_2d': '0', 'zoom_x_2d': '0', 'zoom_y_2d': '0', 'sampling_mode': 'bicubic', 'infill_mode': 'wrap', 'pixel_size': 1, 'smoothing_weight': 1, 'random_initial_palette': True, 'palette_size': 6, 'palettes': 9, 'gamma': 1, 'hdr_weight': 0.1, 'palette_normalization_weight': 0.2, 'show_palette': False, 'target_palette': '', 'lock_palette': False, 'frames_per_second': 20, 'direct_stabilization_weight': '', 'semantic_stabilization_weight': '', 'depth_stabilization_weight': '', 'edge_stabilization_weight': '', 'flow_stabilization_weight': '', 'video_path': '', 'frame_stride': 1, 'reencode_each_frame': False, 'flow_long_term_samples': 1, 'ViTB32': True, 'ViTB16': True, 'ViTL14': False, 'ViTL14_336px': False, 'RN50': True, 'RN101': False, 'RN50x4': False, 'RN50x16': False, 'RN50x64': False, 'use_mmc': False, 'file_namespace': 'sample', 'allow_overwrite': True, 'display_every': 25, 'clear_every': 0, 'display_scale': 1, 'save_every': 25, 'backups': 5, 'show_graphs': True, 'approximate_vram_usage': False, 'use_tensorboard': True, 'models_parent_dir': '.', 'gradient_accumulation_steps': 1, 'device': 0, 'resume': True, 'mmc_models': [], 'base_name': 'sample'}
